iosApp/iosApp/Pages/Onboarding/OnboardingPage.swift

```diff
@@ -0,0 +1,33 @@
+//
+//  OnboardingPage.swift
+//  iosApp
+//
+//  Created by Horn, Melody on 2024-10-25.
+//  Copyright © 2024 MBTA. All rights reserved.
+//
+
+import shared
+import SwiftUI
+
+struct OnboardingPage: View {
+    let screens: [OnboardingScreen]
+    @State var selectedIndex: Int = 0
+    let onFinish: () -> Void
+
+    let inspection = Inspection<Self>()
+
+    var body: some View {
+        OnboardingScreenView(screen: screens[selectedIndex], advance: {
```

Comment:
issue: On a run without voice over turned on, I get an index out of range error when hitting "Get started" from the feedback screen

Reply:
This `advance` logic was working when I still had everything in a `TabView` for the page count view, but I guess I forgot to test it again when I got rid of that, and for some reason the unit test works anyway.